### PR TITLE
feat: add support for terraform v0.15

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: examples/organization/.terraform.lock.hcl [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# This file is maintained automatically by "terraform init".
```
Comment (mariux) [inline examples/organization/.terraform.lock.hcl:1]: Please remove lock file

**Review by mariux — CHANGES_REQUESTED** [review]

File: examples/organization/.terraform.lock.hcl (hunk 1 above)
Comment (soerenmartius) [reply examples/organization/.terraform.lock.hcl]: removed

File: README.md [hunk 2]
```diff
@@ -206,7 +206,7 @@ Copyright &copy; 2020 [Mineiros GmbH][homepage]
 [badge-build]: https://github.com/mineiros-io/terraform-github-organization/workflows/CI/CD%20Pipeline/badge.svg
 [badge-semver]: https://img.shields.io/github/v/tag/mineiros-io/terraform-github-organization.svg?label=latest&sort=semver
 [badge-license]: https://img.shields.io/badge/license-Apache%202.0-brightgreen.svg
-[badge-terraform]: https://img.shields.io/badge/terraform-0.14%20|%200.13%20|%200.12.20+-623CE4.svg?logo=terraform
+[badge-terraform]: https://img.shields.io/badge/terraform-0.15%20|%200.13%20|%200.12.20+-623CE4.svg?logo=terraform
```
Comment (mariux) [inline README.md:209]:
```suggestion
[badge-terraform]: https://img.shields.io/badge/terraform-0.15%20|%200.14%20|%200.13%20|%200.12.20+-623CE4.svg?logo=terraform
```

File: examples/organization/README.md [hunk 3]
```diff
@@ -14,7 +14,8 @@ In Addition it blocks two random users and sets up two projects.
 
 ```hcl
 module "organization" {
-  source = "../.."
+  source  = "mineiros-io/organization/github"
+  version = "~> 0.5.0"
```
Comment (mariux) [inline examples/organization/README.md:18]: good catch ;)

**Review by mariux — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (soerenmartius) [reply README.md]: resolved in ec93e64

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**